service/src/main/java/com/codeforcommunity/processor/BlocksProcessorImpl.java

```diff
@@ -188,12 +187,25 @@ public void seedBlockCompletions(JWTData jwtData, List<BlockSeedingInfo> blockSe
 
     blockSeedingInfos.forEach(
         (bsi) -> {
+          String original =
```

Comment:
What is "original" here?

Reply:
it is the record before UPDATE is called